src/http_protocol.cpp

```diff
@@ -194,7 +247,7 @@ namespace prime_server {
       return true;
     }
     //we ran out
-    partial_buffer.assign(cursor, length);
+    partial_buffer.append(cursor, length);
```

Comment:
this line was stopping us from having requests larger than zmqs internal buffer..  so dumb..